use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -11,9 +11,10 @@ cloud_ui:
 
 # About data tiering
 
-Save on storage costs by tiering rarely used data to a low-cost object storage layer.
+Save on storage costs by tiering rarely used data to a low-cost object storage layer
+on Timescale Cloud.
```

Comment:
I propose to skip this change and add an info box similar to what we have on `/use-timescale/data-tiering/index.md`